hack/make-rules/test-cmd.sh

```diff
@@ -104,6 +106,7 @@ run_kube_apiserver
 run_kube_controller_manager
 create_node
 SUPPORTED_RESOURCES=("*")
+echo "${SUPPORTED_RESOURCES[0]}"
```

Comment:
why is this echoed?

Reply:
SUPPORTED_RESOURCES is not used, so we have to delete or export

Reply:
can we delete these unreferenced ones? If you don't choose, shellcheck looks like this:
```
SC2034: APISERVER_PID appears unused. Verify use (or export if used externally).
For more information: https://www.shellcheck.net/wiki/SC2034
```

Reply:
possibly we should export it instead, there are some functions in other files that read this variable. 
https://cs.k8s.io/?q=SUPPORTED_RESOURCES&i=nope&files=&repos=kubernetes/kubernetes

Reply:
this is one of those tricky things shellcheck can't detect, the only way to know is to trace what gets executed in everypath and see if something does read this.